kornia/enhance/adjust.py

```diff
@@ -6,7 +6,7 @@
 
 from kornia.augmentation.utils import _transform_input3d
```

Comment:
@vribeiro1 this still has to be removed from here

Reply:
Totally forgot! Sorry :P